README.md

```diff
@@ -140,6 +140,63 @@ Now that your analysis is complete, there are a couple things you might want to
 
 For most users, the FOSSA CLI will work out-of-the-box without any configuration. Just get an API key, run `fossa analyze`, and view your results in the FOSSA web application.
 
+<details>
+  <summary>
+    <b>Supported Languages</b>
+  </summary>
+
```

Comment:
Text is from ./docs/references/strategy/README.md

Reply:
Would having a link to the docs be enough?

If we have the same docs in two places - there's a high likely hood one will eventually become stale.

Reply:
✔️ with https://github.com/fossas/fossa-cli/pull/1324/commits/381dcfba4b1f1eff1a5e00b1f0d5b46b3f048a1e